pkg/document/crdt/tree.go

```diff
@@ -831,24 +831,22 @@ func (t *Tree) toTreePos(pos *TreePos) *index.TreePos[*TreeNode] {
 			Offset: 0,
 		}
 	} else {
-		leftSiblingOffset, err := parentNode.IndexTreeNode.FindOffset(leftSiblingNode.IndexTreeNode)
-		if err != nil {
-			return nil
-		}
-
-		offset := leftSiblingOffset + 1
 		if leftSiblingNode.IsText() {
-			offset, err = t.IndexTree.LeftSiblingsSize(parentNode.IndexTreeNode, offset)
+			treePos = &index.TreePos[*TreeNode]{
+				Node:   leftSiblingNode.IndexTreeNode,
+				Offset: leftSiblingNode.IndexTreeNode.PaddedLength(),
+			}
+		} else {
+			leftSiblingOffset, err := parentNode.IndexTreeNode.FindOffset(leftSiblingNode.IndexTreeNode)
 			if err != nil {
-				return nil // NOTE(sejongk): should return error instead?
+				return nil
```

Comment:
Could we skip the return of the error?

Reply:
I think it's safer to return error rather than skip it

let me fix